tidb-cloud/pause-or-resume-tidb-cluster.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Pause or Resume a TiDB Cluster
+summary: Learn how to pause or resume a TiDB cluster.
+---
+
+# Pause or Resume a TiDB Cluster
+
+TiDB Cloud allows you to easily pause and resume a cluster that is not in operation at all times.
+
+Comparing with backup and restore, pausing and resuming a cluster takes less time and keeps your cluster state information (including cluster version, cluster configurations, and TiDB user accounts).
+
+> **Note:**
+>
+> Currently, you cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). Once a Developer Tier cluster remains idle for 24 hours, the cluster hibernates automatically, and you can resume it by [connecting to your cluster](/tidb-cloud/connect-to-tidb-cluster.md) or clicking **Resume** for the cluster in the TiDB Cloud console.
+
+## Prerequisites
```

Comment:
Is it better to change it to "Limitations" and put it at the end of this doc?

Reply:
It depends on when we want users to notice this info, before the pausing, or after the pausing.